cmd/extproc/mainlib/main.go

```diff
@@ -109,12 +113,20 @@ func Main(ctx context.Context, args []string, stderr io.Writer) (err error) {
 		slog.String("configPath", flags.configPath),
 	)
 
-	lis, err := net.Listen(listenAddress(flags.extProcAddr))
+	network, address := listenAddress(flags.extProcAddr)
+	lis, err := net.Listen(network, address)
```

Comment:
I see ext-proc container keep crashing with
```
time=2025-05-26T20:13:36.728Z level=INFO msg="starting external processor" version=dev address=unix:///etc/ai-gateway-extproc-uds/run.sock configPath=/etc/filter-config/filter-config.yaml
2025/05/26 20:13:36 error: failed to listen: listen unix /etc/ai-gateway-extproc-uds/run.sock: bind: address already in use
```
Could we remove all socket files first before start listener ?